internal/services/apimanagement/api_management_data_source_test.go

```diff
@@ -28,6 +28,18 @@ func TestAccDataSourceApiManagement_basic(t *testing.T) {
 	})
 }
 
+func TestAccDataSourceApiManagement_tenantAccess(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_api_management", "test")
+	r := ApiManagementDataSource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.tenantAccess(data),
+			Check:  acceptance.ComposeTestCheckFunc(),
```

Comment:
this isn't checking anything, let's check that we have some values here:

```suggestion
			Check:  acceptance.ComposeTestCheckFunc(
				check.That(data.ResourceName).Key("tenant_access.0.enabled").Exists(),
			),
```

we should also update the `basic` test above to ensure there's no items if this isn't consumption:

> 			check.That(data.ResourceName).Key("tenant_access.#").HasValue("0"),

Reply:
Thanks for you feedback. For the basic test, In fact the sku name is not consumption should have `tenant_access`. So I added `check.That(data.ResourceName).Key("tenant_access.#").HasValue("1")` for it. Please let me know if I missed something. Thank you.